### DISCUSSION TITLE
REMINDER: No in-person class sessions this week (2023-03-21 and 2023-03-23)

### DISCUSSION THREAD

**Discussion by jdmar3** [comp426-2023-spring/syllabus#28] [Announcements]
Hej allihopa! God morgon. (Hi everyone! Good morning.)

This is just a reminder that we are not having in-person class this week because I am in Gothenburg, Sweden at the Research Data Alliance 20th Plenary Meeting representing the Odum Institute and UNC.

Instead of in-person, I am posting links to brief Zoom walkthroughs and links to tutorials with exercises for HTML+CSS (2023-03-21) and Browser-side JavaScript (2023-03-23). So, be on the lookout for those. You can complete them at your leisure. There will be no regular assignment grade associated. You will get extra credit if you submit them through the normal GitHub means.

I will update this post with links when the lectures, etc. are available in the notes page.

I hope you all had a relaxing or at least pleasant spring break and I hope you have a good week this week. 
I look forward to seeing you all back in class next week.

**Comment by mtemple317** [comment]
I'm not seeing these Zoom walkthroughs/ links to tutorials with exercises for HTML, CSS,, and JavaScript. If these have been posted, could someone point me in the right direction? Thank you!
